server.error.include-message=always
spring.datasource.url=jdbc:mysql://${DB_HOST:mysql}:3306/test?allowPublicKeyRetrieval=true&useSSL=false
spring.datasource.username=${DB_USER:user}
spring.datasource.password=${DB_PASSWORD:password}
spring.jpa.hibernate.ddl-auto=update
server.port=0
spring.application.name=authentication
spring.cloud.consul.discovery.register=true
spring.cloud.consul.enabled=false
spring.cloud.consul.host=${CONSUL_HOST:localhost}
spring.cloud.consul.discovery.prefer-ip-address=true
spring.cloud.consul.discovery.instance-id=${spring.cloud.client.hostname}:${spring.application.name}:${random.int[1,999999]}
jwt.authorities.key=roles
jwt.token.validity=900
